cypress/support/commands.js

```diff
@@ -84,6 +83,33 @@ Cypress.Commands.overwrite('request', (originalFn, ...args) => {
   return originalFn(Object.assign({}, defaults, options));
 });
 
+Cypress.Commands.add('cleanUpTests', () => {
+  cy.deleteAllCustomRules();
+  cy.deleteAllDetectors();
+  cy.deleteAllIndices();
+});
+
+Cypress.Commands.add('getTableFirstRow', (selector) => {
+  if (!selector) return cy.get('tbody > tr').first();
+  return cy.get('tbody > tr:first').find(selector);
+});
+
+Cypress.Commands.add('triggerSearchField', (placeholder, text) => {
+  cy.get(`[placeholder="${placeholder}"]`).type(`{selectall}${text}`).trigger('search');
+});
+
+Cypress.Commands.add('waitForPageLoad', (url, { timeout = 10000, contains = null }) => {
+  const fullUrl = `${Cypress.env('opensearch_dashboards')}/app/${PLUGIN_NAME}#/${url}`;
```

Comment:
Should we make `opensearch_dashboards` a constant?

Reply:
@amsiglan  That makes sense, let me do that and update the PR.